src/client/java/teammates/client/scripts/scalabilitytests/InstructorFeedbackResultsPageScalabilityTest.java

```diff
@@ -0,0 +1,70 @@
+package teammates.client.scripts.scalabilitytests;
+
+import org.testng.annotations.Test;
+
+import teammates.client.scripts.util.Stopwatch;
+import teammates.common.util.AppUrl;
+import teammates.common.util.Const;
+import teammates.common.util.Logger;
+import teammates.test.cases.browsertests.BaseUiTestCase;
+import teammates.test.pageobjects.InstructorFeedbackResultsPage;
+
+/**
+ * SUT: {@link Const.ActionURIs#INSTRUCTOR_FEEDBACK_RESULTS_PAGE}.
+ *
+ * <p>Run InstructorFeedbackResultsPageScaleTestDataGenerator.java before running the tests.
+ */
+public class InstructorFeedbackResultsPageScalabilityTest extends BaseUiTestCase {
+
+    private static final String DATA_FOLDER_PATH = "src/client/java/teammates/client/scripts/scalabilitytests/data/";
+    private static final Logger log = Logger.getLogger();
+
+    @Override
+    protected void prepareTestData() {
+        // the actual test data is refreshed before each test method
```

Comment:
I think this is not a good idea. We should be able to upload the data once and reuse the same data until we need to upload it again. Writing large amounts of data cost money and does not affect the objectives of the test.